doc/daikon.texinfo

```diff
@@ -2357,11 +2357,11 @@ Static variables of a class have names of the form
 @cindex pre-state variables
 @code{orig(x)} refers to the value of variable @code{x} upon
 entry to a procedure (because the procedure body might modify the value
-of @code{x}).  These variables appear only at @code{:::EXIT} program
-points.  Typically, @code{orig()} variables do not appear in the trace,
-but are automatically created by Daikon when it matches up
-@code{:::ENTER} and @code{:::EXIT@var{nn}} program points.
-@xref{orig variable example}.
+of @code{x}).  These variables appear only at @code{:::EXIT} and 
+@code{:::THROWCOMBINED} program points.  Typically, @code{orig()} 
```

Comment:
Request: please do not re-fill paragraphs.  When you refill a paragraph, it changes many lines in the diff, making it hard to determine what real changes you made.  It doesn't matter if the .texinfo file (which will be processed by Texinfo to make the manual) doesn't have perfectly filled paragraphs, because only developers will read it -- and those developers will care more that the diffs are small and easy to read.

Could you please un-fill the paragraphs that you reformatted?